aws-rds-dbcluster/src/main/java/software/amazon/rds/dbcluster/BaseHandlerStd.java

```diff
@@ -220,6 +223,28 @@ protected boolean isDBClusterStabilized(
         }
     }
 
+    protected boolean isDBClusterStabilizedAfterRemoveFromGlobalCluster(
```

Comment:
i might suggest using a predicative chain rather than duplicating the available-check, like: `isDBClusterStabilized.and(isRemovedFromGlobalCluster)`